rapidgator.sh

```diff
@@ -661,6 +661,9 @@ rapidgator_upload() {
             -F "file=@$FILE;type=application/octet-stream;filename=$DEST_FILE" \
             "$UP_URL$SESSION_ID&folder_id=$FOLDER_ID") || return
 
+        sleep 10
```

Comment:
don't use `sleep` but `wait`, this is the proper plowshare API to use.
10s for synchronising is huge, could you test with 8s?